### ISSUE TITLE
The exported `prerender` property is not copied to files generated by `sync`

### ISSUE THREAD

**Issue by MellKam** [Alexandre-Fernandez/astro-i18n#30]
My `/pages/index.astro`

```astro
---
import { astroI18n } from "astro-i18n";
astroI18n.init(Astro);

export const prerender = true;
---
...
```

Generated file (`/pages/ua/index.astro`):
```astro
---
import Page from "../index.astro"

const { props } = Astro
// would be nice to have `export const prerender = true;`
---

<Page {...props} />
```

Of course, I can just edit them manually, but it would be great to automatically detect exported properties or to be able to enable copying them.

**Comment by Alexandre-Fernandez** [comment]
Thanks for reporting, I totally forgot about this, I'll add it asap.

**Comment by Alexandre-Fernandez** [comment]
fixed in v1.6.10
